k-distribution/tests/regression-new/map-symbolic-tests-haskell/assignment-2-spec.k

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2019 K Team. All Rights Reserved.
+// generated by gen-tests.sh
+
+requires "map-tests.k"
+
+module ASSIGNMENT-2-SPEC
+    imports MAP-TESTS
+
+    claim
+        <k>
+            assignment ( (MAP:Map X:MyId |-> 1) [ Y:MyId <- 2 ] [ Z:MyId <- 3 ] ) =>
```

Comment:
This claim doesn't test anything.

Reply:
To provide some context, these tests were written how they were (with invalid RHSs), so that we would get the actual output of the backend (rather than having the backend try to prove that the RHS actually is semantically equal to the resulting final state). That way we could make sure that both (i) the simplifications we want were happenning, and (ii) the simplifications we _don't_ want were not happenning.

This was to not trust the java backend map equality checks basically, but I think it makes sense to update the tests for the haskell backend because we trust it's routines more.